### PR TITLE
feat(bootstrap): bootstrap project

### PR DESCRIPTION
**Pull request by VEBERArnaud**
# Description

This PR bootstrap the Kong Admin Bundle Project

---

| Q              | A
| -------------- | ---
| Bug fix ?      | no
| New feature ?  | yes
| BC breaks ?    | no
| Deprecations ? | no
| Tests pass ?   | yes
| Fixed tickets  | ~
| License        | MIT

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+sudo: false
+language: php
+php:
+    - 5.6
+    - 7.0
```
Comment (eko) [inline .travis.yml:10]: What about adding php `7.1` too as it is stable?

**Review by eko — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (VEBERArnaud) [reply .travis.yml:10]: yep, done in 4aebacb

File: src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+     * @return void
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $treeBuilder->root('kong_admin');
+
+        return $treeBuilder;
+    }
+
```
Comment (Prims47) [inline src/DependencyInjection/Configuration.php]: Can you delete this line ? :)

Comment (VEBERArnaud) [reply src/DependencyInjection/Configuration.php]: done in e608847

**Pull request merged**